Sources/Extensions/UITableView/UITableView+Extensions.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright (c) 2017 Touch Instinct
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+
+public extension UITableView {
+
+    /// Adds colored background to the top of the tableview.
+    /// Use when you need a colored bounce area with a color
+    /// different from tableview background
+    func addHeaderBackground(with color: UIColor) {
+        let backgroundFrame = CGRect(x: frame.origin.x, y: -bounds.height,
+                                     width: UIScreen.main.bounds.width,
```

Comment:
`bounds.width`

Reply:
then it should be called only after view laid out subviews, otherwise empty uncoloured space may appear. usually we call this method in addViews() of the ConfigurableController, which is called from viewDidLoad.